ambry-api/src/main/java/com/github/ambry/rest/RestUtils.java

```diff
@@ -258,6 +260,10 @@ public static final class Headers {
      * It is set to a string that differentiate STITCH vs regular upload named blob. If it set to "STITCH", it indicate that this is the a stitch upload.
      */
     public static final String UPLOAD_NAMED_BLOB_MODE = "x-ambry-upload-named-blob-mode";
+    /**
+     * It is set to a string that differentiate STITCH vs regular upload dataset version. If it set to "STITCH", it indicate that this is the a stitch upload.
+     */
+    public static final String UPLOAD_DATASET_VERSION_MODE = "x-ambry-upload-dataset-version-mode";
```

Comment:
We don't really have to create another header for this, we can just reuse the named blob mode.

Reply:
I was planning to avoid involve the named blob related header to avoid confusion for client. It's better that if they could stick to the dataset concept. Let me know if you have different opinion.

Reply:
but we are already using named blob endpoints for the upload, might as well just reuse the headers.

Reply:
sure, removed it.